lib/lotus/view/configuration.rb

```diff
@@ -377,9 +379,39 @@ def duplicate
       def load!
         views.each   { |v| v.__send__(:load!) }
         layouts.each { |l| l.__send__(:load!) }
+        load_partials!
         freeze
       end
 
+      # Load partials for each partial template file found under the
+      # given load paths
+      #
+      # @since x.x.x
+      # @api private
+      def load_partials!
+        Lotus::View::Rendering::PartialTemplatesFinder.find_partials(root).each do |key, format, template|
```

Comment:
@stevehook This set of `key`, `format`, `template` smells like a [Data Clump](http://martinfowler.com/bliki/DataClump.html). Is it reasonable to yield just an object named `Partial`.

This would also simplify the signature of `#add_partial(partial)`.